docs/glossary.md

```diff
@@ -6,58 +6,60 @@ title: Glossary
 
 **Credentials** - Unique values that prove the legitimacy of a user or individual.
 
-**Centralisation** - Refers to the The concentration of control of an activity or organisation under a single authority
+**Centralisation** - Refers to the concentration of control of an activity or organisation under a single authority.
 
-**Delegating NYM** - Similiar to bonding but by non operators and as a proof of their trust on a node operator providing a good quality of service 
+**Delegating NYM** - Similar to bonding but by non-operators and as proof of their trust in a node operator providing a good quality of service. 
 
 **Decentralisation** - Refers to the transfer of control and decision-making from a centralised entity (individual, organization, or group thereof) to a distributed network.
 
 **Encryption** - The process of encoding information.
 
-**[Gateways](https://blog.nymtech.net/gateways-to-privacy-51196005bf5)** - The entrance to the mixnet that encrypted packets must pass through before being forwarded to mixnodes. They also act as a sort of mailbox for messages.
+**[Gateways](https://blog.nymtech.net/gateways-to-privacy-51196005bf5)** - The entrance to the mixnet that encrypted packets must pass through before being forwarded to mix nodes. They also act as a sort of mailbox for messages.
+
+**[Gateway Bonding](https://nymtech.net/docs/stable/run-nym-nodes/nodes/gateways/#bonding-your-gateway)** - A action for node operators to bond their gateway to their Nym Wallet.
+
+**Metadata** - A set of data that describes and gives information about other data such as IP addresses, device types, geolocations etc.
 
-**Metadata** - A set of data that describes and gives information about other data such IP addresses, device types, geolocations etc.
+**[Mixnet](https://nymtech.net/docs/stable/architecture/traffic-flow/)** - Routing protocols that create hard-to-trace communications by using a chain of proxy servers known as mixes which take in messages from multiple senders, shuffle them, and send them back out in random order to the next destination.
 
-**Mixnet** - Routing protocols that create hard-to-trace communications by using a chain of proxy servers known as mixes which take in messages from multiple senders, shuffle them, and send them back out in random order to the next destination.
+**[Mix Nodes](https://blog.nymtech.net/nym-mixnodes-deep-dive-d2b91917f097)** - A individual node within the mixnet where network traffic passes through as part of the infrastructure on the Nym Mixnet.
 
-**[Mix Nodes](https://blog.nymtech.net/nym-mixnodes-deep-dive-d2b91917f097)** - A individual node within the mixnet where network traffic passes through as part of the infrastucture on the Nym Mixnet
+**[Mix node Bonding](https://nymtech.net/docs/stable/run-nym-nodes/nodes/mixnodes/#bonding-your-mix-node)** - A action for node operators to bond their mix node to their Nym Wallet.
```

Comment:
no - bonding is annoucing to the network that this node is ready to route traffic through. It happens mostly _via_ the wallet, but can also be done by the nym-cli too. Check the bonding blogpost for a more fleshed out defnition.